torch_xla/csrc/aten_xla_bridge.cpp

```diff
@@ -131,18 +131,12 @@ void XlaUpdateTensors(absl::Span<const at::Tensor> dest_xla_tensors,
   for (auto index : indices) {
     at::Tensor dest = dest_xla_tensors.at(index);
     at::Tensor source = source_cpu_tensors.at(index);
-    XLATensorImpl* dest_impl = GetXlaTensorImpl(dest);
-    if (dest_impl != nullptr) {
-      auto xla_source = TryGetXlaTensor(source);
-      if (!xla_source) {
-        dest_impl->tensor().UpdateFromTensorOut(source);
-      } else {
-        dest_impl->tensor().UpdateFromTensorOut(*xla_source);
-      }
-      dest_impl->force_refresh_sizes();
-    } else {
-      dest.resize_as_(source).copy_(source);
-    }
+    // When this functions is called in aten_xla_type_default.cpp,
+    // it must be called in either inplace op or out= op since
+    // normal ops doesn't require updating existing tensors.
+    // In both scenarios above resize_as_ doesn't hurt, since either
+    // their sizes already match, or source sizes overrides dest.
+    dest.resize_as_(source).copy_(source);
```

Comment:
I would not do this, as resize_() does not have the break-out-the-view-if-does-not-fit.